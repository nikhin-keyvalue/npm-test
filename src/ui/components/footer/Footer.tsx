import Preact, { h } from 'preact';

import './style.scss';

const Footer: Preact.FunctionComponent = () => (
  <div className='zamp-footer-wrapper'>
    powered by <span className='zamp-footer-name'>ZAMP</span>
  </div>
);

export default Footer;
