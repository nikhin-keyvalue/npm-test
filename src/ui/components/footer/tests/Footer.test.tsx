import { h } from 'preact';
import { configure, mount } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { Adapter } from 'enzyme-adapter-preact-pure';

import Footer from '../Footer';

configure({ adapter: new Adapter() });

describe('Counter', () => {
  it('render components', () => {
    const wrapper = mount(<Footer />);
    const tree = shallowToJson(wrapper);

    expect(tree).toMatchSnapshot();
  });
});
