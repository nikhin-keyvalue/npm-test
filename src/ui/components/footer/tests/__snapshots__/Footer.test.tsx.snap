// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter render components 1`] = `
<Footer>
  <div
    className="zamp-footer-wrapper"
  >
    powered by 
    <span
      className="zamp-footer-name"
    >
      ZAMP
    </span>
  </div>
</Footer>
`;
